### PR TITLE
Adds some missing requires for annotations 

### PR DESCRIPTION
**Pull request by sbrunner**
Required by c2cgeoportal build on Docker

### CODE REVIEW

**Comment by sbrunner** [comment]
Yes ...

File: contribs/gmf/src/services/permalink.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ goog.provide('gmf.Permalink');
 goog.require('gmf');
 goog.require('ngeo');
 goog.require('ngeo.AutoProjection');
+/** @suppress {extraRequire} */
+goog.require('gmf.ObjectEditingManager');
```
Comment (gberaudo) [inline contribs/gmf/src/services/permalink.js]: Ideally you should not require `gmf.ObjectEditingManager` here since it is an optionnal dependency.

Comment (sbrunner) [inline contribs/gmf/src/services/permalink.js]: :-) then where, If I add it in the gmf.js, it's a requirement for all the module...

Comment (sbrunner) [inline contribs/gmf/src/services/permalink.js]: For me it's an issue of closure-utils ...

**Pull request merged**